### DISCUSSION TITLE
API : CreateTable to include ID, CreatedAt, UpdatedAt [Apr 16]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5524] [Discord Threads]
> 🙋‍♂️ **Rahim Dobani**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### ```   
#### tableCreate: {   
####         method: 'POST',   
####         url: 'http://localhost:8080/api/v1/db/meta/projects/p_k3wjewocvyj7wc/tables',   
####         data: {   
####             table_name: 'sampleTable5',   
####             columns: [   
####                 {   
####                     title: 'Id',   
####                     column_name: 'id',   
####                     uidt: "ID",   
####                 },   
####                 {   
####                     title: 'Number',   
####                     column_name: 'number',   
####                     uidt: "Number",   
####                 },   
####                 {   
####                     title: 'CreatedAt',   
####                     column_name: 'created_at',   
####                     uidt: "DateTime",   
####                        
####                     // column default value   
####                     cdf: 'now()',   
####                 },   
####                 {   
####                     title: 'UpdatedAt',   
####                     column_name: 'updated_at',   
####                     uidt: "DateTime",   
####                        
####                     // column default value   
####                     cdf: 'now()',   
####                     // auto increment   
####                     au: 1,   
####                 }   
####             ]   
####         }   
####     }   
#### ```  
#  
> 🙋‍♂️ **Rahim Dobani**  
#### CreatedAt also updating in this json form when updating data using patch api. Is there any solution only updatedAt updated when updating data?  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### double check if you included `au` in created at also? i didn't notice that issue though when i tried locally  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1085533029272924291)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
